src/utils/debounce.ts

```diff
@@ -0,0 +1,26 @@
+
+export function debounce<T extends (...args: any[]) => void>(fn: T, wait?: number, leading?: boolean) {
+  let timer: number | NodeJS.Timeout;
```

Comment:
I don't think our environment includes node.js. Any reason you add `NodeJS.Timeout` here? 🤔